### ISSUE TITLE
data generation .....  attaching a datasource to a flowop?

### ISSUE THREAD

**Issue by ea69** [filebench/filebench#126]
The Spring 2016 ;login: article has a section on Advanced Features which are said to include data generation. There is mention of a _datasource_ that can be attached to any flowop in order to test data reduction technologies rather than always writing zeros or arbitrary data.

But, I cannot find any references to this feature in the WML reference. It is not used in any of the examples or predefined personalities. The term _datasource_ does not even get a hit if I grep for it in the source tree.

Am I missing something obvious?

**Comment by sectorsize512** [comment]
Yeah, there was some development on that side but it never got into mainline, unfortunately.
